crypto/pkcs11key/key.go

```diff
@@ -1,4 +1,4 @@
-// +build !nopkcs11
+// +build pkcs11
```

Comment:
Shouldn't need a build tag. In general, the end state of this change should be that there are no more build tags, and `pkcs11key` is a standalone package that has no dependencies into or out of the other packages.